testsuite/integration/manualmode/src/test/java/org/wildfly/test/manual/observability/opentelemetry/ContextPropagationTestCase.java

```diff
@@ -151,7 +145,8 @@ public void testContextPropagation() throws Exception {
                 TestSuiteEnvironment.getServerAddress(), TestSuiteEnvironment.getServerPort(), "remote+http");
 
         URL serviceUrl = new URL(managementClient.getWebUri().toURL(), '/' + DEPLOYMENTA);
-        Response response = ClientBuilder.newClient()
+        Client client = ClientBuilder.newClient();
```

Comment:
If closing is not part of the test: try/finally or try with resources?

Reply:
Updated with try-with-resources already.